db/migrate/20240902115121_remove_pet_adopter_foster_account_index_from_adopter_applications.rb

```diff
@@ -0,0 +1,5 @@
+class RemovePetAdopterFosterAccountIndexFromAdopterApplications < ActiveRecord::Migration[7.1]
```

Comment:
Do you think we should replace this index with form_submission_id instead of completely removing it?